lib/Common/Codex/Utf8Helper.h

```diff
@@ -69,8 +75,10 @@ namespace utf8
         }
 
         utf8::DecodeIntoAndNullTerminate(destString, (LPCUTF8) sourceString, cchDestString);
+        Assert(destString[cchDestString] == 0);
```

Comment:
This looks like redundant. `DecodeIntoAndNullTerminate` puts `0` there no matter what.